### DISCUSSION TITLE
Welcome to remark42 Discussions!

### DISCUSSION THREAD

**Discussion by umputun** [umputun/remark42#831] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kostimoshenko** [comment]
Hello guys

I tried to make telegram authentication work yesterday but failed to do so. I did create a bot and followed the other instructions listed in the readme file. The login form had shown a new field for telegram username, but the button to proceed with the login has not done anything. Given that I am not a developer I leave a chance that I did something wrong, but I though I would ask anyway: is this feature supposed to work already?

**Reply by paskal** [reply]
It's not ready yet, unfortunately: @akellbl4 still need to enable it in the frontend.

**Reply by kostimoshenko** [reply]
Thanks, it now makes sense why I couldn't make it work 🙂
If that's not too much to ask for, could you please also tell me how do I find out when this feature is activated?
